data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -297,7 +297,7 @@
         [message]
             speaker=narrator
             image="units/infantry-commander.png~RC(magenta>red)"
-            message= _ "Sir Gerrick is an infantry commander. He is a strong warrior in his own right, but when he is near less experienced troops, he makes their attack better too!"
+            message= _ "As a senior infantry officer, Sir Gerrick is a powerful and seasoned warrior. He raises the fighting spirit of less experienced troops near him and causes their attacks to deal more damage."
```

Comment:
Better, "As a senior infantry officer, Sir Gerrick is a powerful warrior in his own right, but also raises the fighting spirit of less-experienced troops near him and causes their attacks to deal more damage."